<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VuokraEmititForm</class>
 <widget class="QWidget" name="VuokraEmititForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tulospalvelu - Vuorka-Emitit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Emitit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="naytaKaikkiBox">
       <property name="text">
        <string>N채yt채 kaikki emitit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="emitView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="poistaButton">
       <property name="text">
        <string>Poista</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Emit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numeroBox">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="vuosiBox">
       <property name="prefix">
        <string>vuosi </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="kuukausiBox">
       <property name="prefix">
        <string>kk </string>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lisaaButton">
       <property name="text">
        <string>&amp;Lis채채</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Sulje</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
